@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -102,7 +101,6 @@ export interface PickResult {
  * Handles the picking of scene geometry and roads.
  */
 export class PickHandler {
-    private readonly m_rayCaster: PickingRaycaster;
     private readonly m_plane = new THREE.Plane(new THREE.Vector3(0, 0, 1));
```

Comment:
If PickingRayCaster is no longer required, please remove file.

Reply:
The PickingRaycaster is used for MapViewPoints that used for Circles and Squares classes. They are almost unused in the code but I don't really sure that I can remove them.